src/Microsoft.TestPlatform.Extensions.TrxLogger/Utility/Constants.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.Extensions.TrxLogger.Utility
+{
+    using System;
+    using Microsoft.TestPlatform.Extensions.TrxLogger.ObjectModel;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
+    internal static class Constants
+    {
+        /// <summary>
+        /// Uri used to uniquely identify the TRX logger.
+        /// </summary>
+        public const string ExtensionUri = "logger://Microsoft/TestPlatform/TrxLogger/v1";
```

Comment:
>public [](start = 8, length = 6)

nit: all these const should be near to the consumers. will be easy to read and understand